README.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+Build two websites for two local organizations: Funny Bone and Ethan Stark.  Deploy them.
+
+## Specifications
+
+### Funny Bone
+- [ ] Implement a e-commerce application
+  - [ ] Administrator view is separate from visitors view
+  - [ ] Implement OAUTH using GMAIL for administrator access (as an administrator, i can log into the website)
+  - [ ] As an administrator, I can add inventory
+  - [ ] As an administrator, I can delete inventory
+  - [ ] As an administrator, I can update inventory
+  - [ ] As a an administrator, I can search for inventory
+  - [ ] As a visitor, I can view store's inventory
+  - [ ] As a visitor, I can send administrator inquiries about specific inventory
+  - [ ] As a visitor, I can browse inventory by category
+- [ ] Deploy to Heroku
+
+### Ethan Stark
+- [ ] Build a personal portfolio site showcasing web development projects
+- [ ] As a visitor, I can easily contact Ethan Stark via email or phone
+- [ ] As a visitor, I can navigate to related links: Github, project sites, LinkedIn
+- [ ] As a visitor, I can browse a gallery of web projects
+- [ ] As a visitor, I can click a web project and view additional details
+- [ ] Site is deployed to EthanStark.com
+
+
+## Quality Rubric
+
+- [ ] Variables, functions, css classes, etc. are meaningfully named (no comments exist in code to explain functionality - the names serve that function)
+- [ ] Commit messages are concise and descriptive.
+- [ ] Each commit has a specific scope, and does not include unrelated code.
+- [ ] All features are added via pull requests.
+- [ ] Every pull request has a description summarizing the changes made.
+- [ ] Every pull request has been reviewed with [Prrr](http://prrr.apps.learnersguild.org/).
+- [ ] Code is sufficiently covered with a unit test suite
+- [ ] Displays nicely on a cell phone
```

Comment:
Can you clarify 'nicely'? Should everything be contained with no horizontal scrolllbars, for example? Should there be a media query that shows a hamburger menu when the device height and width are smaller than some threshold?